src/pages/settings/PasswordPage.js

```diff
@@ -1,25 +1,158 @@
+                                <Text style={styles.mb1}>Current Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.currentPassword}
+                                    onChangeText={currentPassword => this.setState({currentPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={styles.mb1}>New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.newPassword}
+                                    onChangeText={newPassword => this.setState({newPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={styles.mb1}>Confirm New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.confirmNewPassword}
+                                    onChangeText={confirmNewPassword => this.setState({confirmNewPassword})}
+                                />
+                            </View>
+                            {this.props.account.requiresTwoFactorAuth ? (
+                                <View style={styles.mb6}>
+                                    <Text style={styles.mb1}>Two Factor Code</Text>
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.twoFactorCode}
+                                        onChangeText={twoFactorCode => this.setState({twoFactorCode})}
+                                        placeholder="Required when 2FA is enabled"
+                                    />
+                                </View>
+                            ) : null}
+                            {!isEmpty(this.props.account.error) && (
+                                <Text style={styles.formError}>
+                                    {this.props.account.error}
+                                </Text>
+                            )}
```

Comment:
>There are a few errors you can get back from the API, each with user-friendly response messages that you could display. The main error codes to look out for are 401 if the old password is wrong and 402 if the password doesn't meet the validation conditions mentioned above.

@NikkiWines Just making sure this is correct but am I just displaying any error message that comes back or is the ask to override the 401's and display `Incorrect password. Please try again.`?

Reply:
Yeah, since `response.message` is pretty user-friendly I think it's fine to just display the error message returned from the API.

eg. With an incorrect old password.
![image](https://user-images.githubusercontent.com/16747822/111195427-78d9aa80-8579-11eb-86aa-8874e3691b21.png)

However, maybe it's a good idea to do something like the following (in User.js in `changePassword()`) to make sure we always have an error message incase there's some weird case where we don't get one back from the API.
```Javascript 
errorMessage = lodashGet(response, 'message', 'Unable to change password');